RELEASE-NOTES.txt

```diff
@@ -1,6 +1,11 @@
 Unreleased
 ---
 
+1.62.1
+------
+* [**] Image block: fix height and border regression. [https://github.com/WordPress/gutenberg/pull/34957]
+* [**] Column block: fix width attribute flout cutoff. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921]
```

Comment:
@enejb I noticed that this entry is referencing a different PR and found a typo:
- `Column block: fix width attribute flout cutoff.` => `Column block: fix width float attribute cut off.`
- `https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921` => `https://github.com/WordPress/gutenberg/pull/35061`

Reply:
Heads up that I'm addressing this issue in https://github.com/WordPress/gutenberg/pull/35432.